src/GafferImage/ImageTransform.cpp

```diff
@@ -231,12 +338,29 @@ void ImageTransform::compute( ValuePlug *output, const Context *context ) const
 		static_cast<M33fPlug *>( output )->setValue( resampleMatrix );
 		return;
 	}
+	else if( output == outTransformPlug() )
+	{
+		M33f result = inTransformPlug()->getValue();
+		if( enabledPlug()->getValue() )
+		{
+			result *= transformPlug()->matrix();
+		}
+		static_cast<M33fPlug *>( output )->setValue( result );
+		return;
+	}
 
 	ImageProcessor::compute( output, context );
 }
 
 void ImageTransform::hashDataWindow( const GafferImage::ImagePlug *parent, const Gaffer::Context *context, IECore::MurmurHash &h ) const
 {
+	ChainingScope chainingScope( context, this );
+	if( chainingScope.chained() )
```

Comment:
We don't need to use the ChainingScope for the outTransform because if the outTransform is used at all, we know it's because it has been chained. We never need a pass-through for the outTransform.